packages/editor/src/components/collab-sidebar/add-comment.js

```diff
@@ -35,25 +35,22 @@ export function AddComment( {
 	// State to manage the comment thread.
 	const [ inputComment, setInputComment ] = useState( '' );
 
-	const {
-		defaultAvatar,
-		clientId,
-		blockCommentId,
-		showAddCommentBoard,
-		currentUser,
-	} = useSelect( ( select ) => {
-		const { getSettings } = select( blockEditorStore );
-		const { __experimentalDiscussionSettings } = getSettings();
-		const selectedBlock = select( blockEditorStore ).getSelectedBlock();
-		const userData = select( coreStore ).getCurrentUser();
-		return {
-			defaultAvatar: __experimentalDiscussionSettings?.avatarURL,
-			clientId: selectedBlock?.clientId,
-			blockCommentId: selectedBlock?.attributes?.blockCommentId,
-			showAddCommentBoard: showCommentBoard,
```

Comment:
Not sure why this prop was passed around in `mapSelect`. It shouldn't be required.